src/main/java/edu/ie3/util/quantities/QuantityUtil.java

```diff
@@ -131,42 +191,89 @@ public static boolean quantityIsEmpty(Quantity<?> that) {
    * @return true, if both are an EmptyQuantity or if they are equal
    * @throws NullPointerException if any quantity is null
    */
-  public static <Q extends Quantity<Q>> boolean isTheSameConsideringEmpty(
-      Quantity<Q> a, Quantity<Q> b) {
+  public static <Q extends Quantity<Q>> boolean equals(Quantity<Q> a, Quantity<Q> b) {
+    return equals(a, b, DEFAULT_TOLERANCE);
+  }
+
+  /**
+   * Compares two {@link Quantity}s, if they are equal. Returns true only if both quantities are of
+   * type {@link EmptyQuantity} or if they are equal in value <b>and</b> Unit. The type of the value
+   * does not matter. Throws a NullPointerException if any quantity is null, as null is not to be
+   * expected as any known empty value should be replaced by an EmptyQuantity.
+   *
+   * <p>isTheSameConsideringEmpty(1 km, 1000 m) // false <br>
+   * isTheSameConsideringEmpty(1.0 km, 1 km) //true
```

Comment:
```suggestion
   * <p>equals(1 km, 1000 m) // false <br>
   * equals(1.0 km, 1 km) //true
```

Additionally, please provide an example, that is actually taking the tolerance argument and also utilizes it.